open-sphere-plugins/feature-actions/src/main/java/io/opensphere/featureactions/controller/FeatureActionsController.java

```diff
@@ -101,36 +95,37 @@ public FeatureActionsController(Toolbox toolbox, FeatureActionsRegistry registry
         addService(myPrefs.getPreferences(StyleManagerController.class).getListenerService(Preferences.ALL_KEY,
                 this::handleStyleChange));
 
-        /* This is needed because handleStyleChange() doesn't handle when 'Enable Custom Style' is checked. */
-        myStyleRegistryListener = new VisualizationStyleRegistryChangeListener()
-        {
-            @Override
-            public void visualizationStyleInstalled(Class<? extends VisualizationStyle> styleClass, Object source)
-            {
-            }
-
-            @Override
-            public void visualizationStyleDatatypeChanged(VisualizationStyleDatatypeChangeEvent evt)
-            {
-                if (!evt.isNewIsDefaultStyle())
-                {
-                    myProcrastinatingExecutor.execute(() -> handleModelChange(evt.getDTIKey()));
-                }
-            }
-
-            @Override
-            public void defaultStyleChanged(Class<? extends VisualizationSupport> mgsClass,
-                    Class<? extends VisualizationStyle> styleClass, Object source)
-            {
-            }
-        };
+        /* This is needed because handleStyleChange() doesn't handle when
+         * 'Enable Custom Style' is checked. */
+//        myStyleRegistryListener = new VisualizationStyleRegistryChangeListener()
+//        {
+//            @Override
+//            public void visualizationStyleInstalled(Class<? extends VisualizationStyle> styleClass, Object source)
+//            {
+//            }
+//
+//            @Override
+//            public void visualizationStyleDatatypeChanged(VisualizationStyleDatatypeChangeEvent evt)
+//            {
+//                if (!evt.isNewIsDefaultStyle())
+//                {
+//                    myProcrastinatingExecutor.execute(() -> handleModelChange(evt.getDTIKey()));
+//                }
+//            }
+//
+//            @Override
+//            public void defaultStyleChanged(Class<? extends VisualizationSupport> mgsClass,
+//                    Class<? extends VisualizationStyle> styleClass, Object source)
+//            {
+//            }
+//        };
     }
 
     @Override
     public void open()

```

Comment:
do you need to ovverride this since you can just inherit it. Also just read you meant to leave the comments so i guess this needs to stay just in case